noxfile.py

```diff
@@ -213,14 +167,9 @@ def isort(session: nox.sessions.Session):
 
     """
     # Pip install the session requirements.
-    session.install("isort")
-    # Execute the isort import checker on the package.
-    session.run("isort", "--check", str(PACKAGE))
-    # Execute the isort import checker on the documentation.
-    session.run("isort", "--check", str(ROOT_DIR / "docs"))
-    # Execute the isort import checker on the root Python files.
-    for fname in ROOT_DIR.glob("*.py"):
-        session.run("isort", "--check", str(fname))
+    session.install("pre-commit")
+    # Execute the pre-commit linting tasks.
+    session.run("pre-commit", "run", "--all-files")
```

Comment:
```suggestion
    args = ["pre-commit", "run", "--all-files"]
    hooks = ["black", "flake8", "isort"]
    for hook in hooks:
        session.run(*args, hook)
    
```

I've just discovered with cf-units that this doesn't work well since it's also running the various non-linting hooks, which leads to a failure post-merge ("don't commit to main branch").

Reply:
Nice. It's a shame you can't do something like `pre-commit list`... I'll dig :+1:

Reply:
Shame. There isn't anything obvious... which I'm surprised by.

Reply:
Happy to go with this... and I've updated it to include the `blacken-docs` hook